app/renderer/components/main/braveryPanel.js

```diff
@@ -935,14 +935,18 @@ const styles = StyleSheet.create({
     marginBottom: '.75rem'
   },
   braveryPanel_compact__body__ul: {
-    padding: '0 .5rem',
-    margin: '0 0 .75rem 0',
-    wordBreak: 'break-all',
+    padding: '0 .5rem 1rem',
+    margin: '0 0 .25rem 0',
```

Comment:
These are set to make the list scrollable (y axis) to its bottom. I could not find out how to do the same thing for x axis.